### PR TITLE
Implemented week weather display functionality

### PR DESCRIPTION
**Pull request by cpfergus1**
Co-authored-by: GeorgeSoderholm <62966396+GeorgieGirl24@users.noreply.github.com>

- [x] Updated Files:

- [x] Repo Representation: Front End

- [x] All Tests are passing

- [ ] Updated README.md with added information

- [x] Collabotors: George

Finished Styling the flight show page... huzzah!
Screen Shot 2021-01-13 at 4 11 48 PM

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -139,9 +141,9 @@ GEM
       oauth2 (~> 1.1)
       omniauth (>= 1.1.1)
       omniauth-oauth2 (>= 1.6)
-    omniauth-oauth2 (1.7.0)
+    omniauth-oauth2 (1.7.1)
       oauth2 (~> 1.4)
-      omniauth (~> 1.9)
```
Comment (stillsheryl) [inline Gemfile.lock:144]: You may want to double-check that we're not pushing up this. I think we want to keep it as `omniauth (~> 1.9)`

**Review by Todd-Estes — COMMENTED** [review]
Awesome!

File: Gemfile.lock (hunk 1 above)
Comment (cpfergus1) [reply Gemfile.lock:144]: I believe it was an issue when oauth was version 2.0 please correct me if I am wrong but all test passed

**Review by dunlapww — APPROVED** [review]

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -357,6 +386,31 @@ h3 {
...
+  opacity: 100%;
+}
+.myButton:hover {
+	background:linear-gradient(to bottom, #bab1ba 5%, #ededed 100%);
+	background-color:#bab1ba;
+}
+.myButton:active {
+	position:relative;
+	top:1px;
+}
```
Comment (JesseMellinger) [inline app/assets/stylesheets/application.scss:413]: Nice use of `hover` and `active` here! Styling of the buttons looks great.

**Review by TeknoServal — APPROVED** [review]

File: app/views/flights/show.html.erb [hunk 3]
```diff
@@ -32,19 +32,29 @@
...
+      <div class="weather-image">
+        <%= image_tag(day.image_path, class: "weather-icon") %>
+        <div class="weather-temp">
+          <p><%= day.date.to_date.strftime("%m/%d/%Y")%></p>
+          <p>Feels like: <%= day.day_feels_like_f.round(0)%>°</p>
+          <p>Hi: <%= day.max_f.round(0)%>° Lo: <%=day.min_f.round(0)%>°</p>
+        </div>
+      </div>
+    </div>
+    <% end %>
```
Comment (JesseMellinger) [inline app/views/flights/show.html.erb:50]: Iterating over the forecast array and using the image path on the `WeatherDay` objects. Nice!

**Review by JesseMellinger — APPROVED** [review]

**Pull request merged**